redfish-core/lib/log_services.hpp

```diff
@@ -4082,6 +4070,30 @@ inline void handleLogServicesDumpEntryComputerSystemDelete(
     deleteDumpEntry(asyncResp, dumpEntryId, "System");
 }
 
+inline void handleLogServicesDumpEntryComputerSystemDownloadGet(
+    crow::App& app, const std::string& dumpType, const crow::Request& req,
+    const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+    const std::string& /*dumpId*/)
+{
+    if (!redfish::setUpRedfishRoute(app, req, asyncResp))
+    {
+        return;
+    }
+    std::string url(req.target());
+    std::string startDelimiter = "Entries/";
+    std::size_t pos1 = url.rfind(startDelimiter);
```

Comment:
can we use url.segments() to split urls in to array of string and use std::find to find segment after the Entries segment?

Reply:
Done